acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ThreadBesuNodeRunner.java

```diff
@@ -413,9 +415,16 @@ TransactionSimulationServiceImpl provideTransactionSimulationService(
       return retval;
     }
 
+    @Provides
+    KeyValueStorageFactory provideKeyValueStorageFactory() {
+      return toProvide
+          .getStorageFactory()
+          .orElse(new InMemoryStoragePlugin.InMemoryKeyValueStorageFactory("memory"));
+    }
+
     @Provides
     @Singleton
-    MetricCategoryRegistryImpl provideMetricCategoryRegistry() {
+    MetricCategoryRegistryImpl provide() {
```

Comment:
Why did you rename this? The previous was name was more descriptive

Reply:
By mistake. I'll fix it, thanks!